### ISSUE TITLE
运行环境

### ISSUE THREAD

**Issue by simba2icac** [elvis2els/mapproject#1]
作者，您好，我下载了您的代码，在我Ubuntu上运行时出错了  请问能大概说一下程序的运行环境么？万分谢谢！

**Comment by elvis2els** [comment]
这个我就是瞎写着玩的，这个版本能不能编译过我都不知道。。。我用的linux mint18。roadnetworkkit那个项目的是可以编译过运行的

**Comment by simba2icac** [comment]
谢谢你啊

**Comment by simba2icac** [comment]
roadnetworkkit这个项目我卡在
--2017-04-20 04:20:29--  http://syh-tj-ubuntu.local/boost_1_57_0.tar.bz2
Resolving syh-tj-ubuntu.local (syh-tj-ubuntu.local)... failed: Name or service not known.
wget: unable to resolve host address ‘syh-tj-ubuntu.local’

我也试着修改了 /etc/resolv.conf  文件  但是报上述的错误，请问您之前遇到过这个问题么？或者知道如何解决么？调试了很久了，希望能得到您的帮助。。。谢谢！

**Comment by elvis2els** [comment]
这个错误我没遇到过，但是 syh-tj-ubuntu.local 是指向的原作者的主机名，这个肯定是不对的。你检查一下这个是在什么文件里报出来的错，按说不应该会有原作者主机名的东西出现

**Comment by simba2icac** [comment]
我第一次在linux下调试程序，所以我运行了 setup_ubuntu.sh这个文件，然后就出现了上面的问题。

**Comment by elvis2els** [comment]
你在根目录下mkdir build
cd build
cmake ..
make
用的是CMakeList来编译的

**Comment by simba2icac** [comment]
ok.....谢谢，，提示我
Unable to find the requested Boost libraries.

  Boost version: 1.53.0

  Boost include path: /usr/include

  The following Boost libraries could not be found:

          boost_coroutine

我装好 libboost1.54-all-dev 应该可以继续吧


真是太感谢了啊  您之前的项目有的 这个步骤  我这太水了

**Comment by simba2icac** [comment]
为啥下载以后sutil文件夹里面没有文件 然后我将sutil @ 6fb077b文件夹中的文件复制粘贴到sutil后 报了一堆error ,这是什么情况呢？谢谢！！！

**Comment by elvis2els** [comment]
你单独下载一下，那个是附加文件夹

**Comment by simba2icac** [comment]
我单独下载了，因为clone以后sutil里面没有文件，然后我将roadnetworkkit项目中sutil @ 6fb077b下载下来，解压后将 boost文件夹，cow_pimpl.hpp等文件复制到了sutil文件下 ，然后就报错了

**Comment by elvis2els** [comment]
boost解压后是需要安装的，sutil里的那些文件只是因为修改了boost里的一些文件

**Comment by simba2icac** [comment]
对 我安装了 boost 我安装了 用的是 sudo apt-get install libboost1.54-all-dev,sutil @ 6fb077b下载解压后也有一个boost文件夹，然后错误如下：
Scanning dependencies of target roadnetworkkit
[  4%] Building CXX object CMakeFiles/roadnetworkkit.dir/src/generalmap.cpp.o
[  8%] Building CXX object CMakeFiles/roadnetworkkit.dir/src/roadmap.cpp.o
In file included from /home/aboutyun/roadnetworkkit/src/util.h:9:0,
                 from /home/aboutyun/roadnetworkkit/src/roadmap.cpp:11:
/home/aboutyun/roadnetworkkit/src/sutil/boost/geometry/distance_projected_point_return_point.hpp:212:90: error: wrong number of template arguments (3, should be 1)

**Comment by elvis2els** [comment]
这个不清楚啊，根据报错是说参数数量错误，我这边编译是正常的

**Comment by simba2icac** [comment]
能给我传一份你的程序吗，只做学术研究，我是东北大学计算机学院的研究生，邮箱：1570952@stu.neu.edu.cn

**Comment by simba2icac** [comment]
谢谢大神，还是一样的错误  ，我要哭了 ，我在看看我哪有问题吧，今天太麻烦你了，特别感谢！！！

**Comment by elvis2els** [comment]
你去boost官网下个最新版的安装试试

**Comment by simba2icac** [comment]
好！

**Comment by simba2icac** [comment]
可以了   谢谢龙哥

**Comment by simba2icac** [comment]
可以了  谢谢大神
